Content.Shared/Mousetrap/MousetrapVisuals.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Mousetrap;
+
+[Serializable, NetSerializable]
+public enum MousetrapVisuals
+{
+    Visual,
+    Armed,
+    Unarmed
+}
```

Comment:
kind of weird that these are part of the same enum but uhhh sure i guess it works